Writing Automated Tests/Tests/Intro/task-info.yaml

```diff
@@ -1,8 +1,6 @@
 type: theory
 files:
-- name: Cargo.toml
-  visible: false
 - name: src/main.rs
   visible: true
-- name: src/lib.rs
+- name: Cargo.toml
```

Comment:
It is visible in this task, but is not visible in others.
I guess the initial approach was to leave it hanging, for the learners to get used to it, as it is a necessary part of a rust program. I believe we should make it one way or another, but consistent across the course.